packages/ember-data/lib/system/store/serializer-response.js

```diff
@@ -3,6 +3,35 @@ import Model from 'ember-data/system/model/model';
 const get = Ember.get;
 
 /**
+  This is a helper method that validates a JSON API top-level document
+
+  The format of a document is described here:
+  http://jsonapi.org/format/#document-top-level
+
+  @method isValidJsonApiDocument
```

Comment:
I'm fine with anything, but my initial thought was to ensure there was no confusion having to do with application-level validation logic (or validation error messages). I'll change it as you suggest